common/src/java/com/zimbra/common/soap/AdminConstants.java

```diff
@@ -1303,6 +1304,13 @@ public final class AdminConstants {
     public static final String A_VOLUME_USE_IN_FREQ_ACCESS = "useInFrequentAccess";
     public static final String A_VOLUME_USE_IN_FREQ_ACCESS_THRESHOLD = "useInFrequentAccessThreshold";
     public static final String A_VOLUME_USE_INTELLIGENT_TIERING = "useIntelligentTiering";
+    public static final String A_VOLUME_URL = "url";
+    public static final String A_VOLUME_ACCOUNT = "account";
+    public static final String A_VOLUME_NAMESPACE = "namespace";
```

Comment:
namespace should be nameSpace as it is taking value from zimbraServerExternalStoreConfig where in key **nameSpace** is provided.

Reply:
done